f/connectors/alerts/alerts_gcs.py

```diff
@@ -423,37 +472,41 @@ def _create_alerts_table(self, table_name):
             cursor.execute(query)
             conn.commit()
 
-    def handle_output(self, geojsons, metadatas):
+    def handle_output(self, alerts, alerts_metadata):
         """
-        Processes GeoJSON data/metadata from Dagster assets and inserts it into a PostgreSQL database. It iterates over each GeoJSON/metadata object, extracts relevant features and properties, and constructs SQL queries to insert these data into the database. After processing all features, it commits the transaction and closes the database connection.
+        Inserts alerts and metadata from GeoJSON data into a PostgreSQL database.
+
+        This method processes a list of alerts and their corresponding metadata, extracting relevant features and properties from each GeoJSON object. It constructs and executes SQL queries to insert these data into the appropriate database tables. The method ensures that each alert and metadata entry is processed within a transaction, committing the transaction upon successful insertion or rolling back in case of errors.
+
+        Parameters
+        ----------
+        alerts (list): A list of alert objects, each containing GeoJSON data with features and properties to be inserted into the database.
+        alerts_metadata (list): A list of metadata objects associated with the alerts, containing additional information to be stored in a separate metadata table.
         """
         table_name = self.table_name
         conn = self._get_conn()
         cursor = conn.cursor()
 
         try:
-            if geojsons:
-                # check to see if we need to create the table for the first time
-                # FIXME: this func creates its own cursor but we already have one
```

Comment:
yeah, cursors are extremely lightweight and it's common to create new ones all the time.  I see no problem with various functions creating (totally managing) their own.